deploy/helm/flightctl/templates/flightctl-prometheus-service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
We should not install this on OCP, i.e.

{{ if .Values.prometheus.enabled }}
...
...